test/unit/utils.py

```diff
@@ -22,26 +17,25 @@
 SERVER_SECRET_KEY = "JTKVSB%%)wK0E.X)V>+}o?pNmC{O&4W4b!Ni{Lh6"
 
 
-@contextmanager
-def run_rpcq_server(server: rpcq.Server, port: int):
-    def run_server():
-        server.run(endpoint=f"tcp://*:{port}", loop=asyncio.new_event_loop())
-
-    @retry(tries=10, delay=1.0)
-    def check_connection():
-        s = socket()
-        try:
-            s.connect(("localhost", port))
-        finally:
-            s.close()
-
-    proc = Process(target=run_server)
-    try:
-        proc.start()
-        check_connection()
-        yield
-    finally:
-        os.kill(proc.pid, signal.SIGINT)
+def patch_rpc_client(*, mocker: MockerFixture, return_value: Any) -> MagicMock:
```

Comment:
Since this is patch `rpcq.Client` I think we should call it `patch_rpcq_client`.

Reply:
Agreed